wordpress-mobile/WordPress-Android
# Issue/9815 domain registration analytics

Fixes #9815

This PR adds tracking events for the domain registration process.

To test:

- Install the wasabi build.
- Make sure the domain registration related events are fired (or will probably fire in case of error).


Update release notes:
